src/redshift-adjust

```diff
@@ -42,10 +42,14 @@ EOF
 fi
 
 if [ ! -e "${file_temp}" ]; then
-    echo 6500 > "${file_temp}"
+    temperature=$(redshift -p | grep -Pzo "(?<=Color temperature: ).*")
```

Comment:
`-P`, `-z`, and `-o` are not in grep(1posix), can you use `sed` instead?

Reply:
I am trying, but I cannot find how to get only the content of where the line is using only `sed`. Would
```shell-script
temperature=$(redshift -p | grep 'Brightness:' | sed -e 's/Brightness: \(.*\)/\1/')
```
suffice? If not, any idea?

Reply:
temperature=$(redshift -p | sed -n 's/Brightness: \(.*\)/\1/p')